ietf-optical-impairment-topology.tree

```diff
@@ -256,6 +260,14 @@ module: ietf-optical-impairment-topology
             /tet:local-link-connectivities:
     +--ro add-path-impairments?    leafref
     +--ro drop-path-impairments?   leafref
+    +--ro transceiver* []
```

Comment:
Does this list need to be keyed with transponder-ref+transceiver-ref? If there is a scenario that one needs to look up the optical impairment of a specific transceiver then it perhaps needs a key.